rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
+                function(cb){
+                  setEdge(pinForSetup, edge, cb);
+                },
+                function(err){
+                  // wrapped here because waterfall can't handle positive result
```

Comment:
probably not a great explanation in the code.

I would have liked to simply use _next_ instead of the anonymous wrapper function. However, waterfall doesn't expect the normal err and result to its next functions. If a second parameter is provided, the waterfall fails. However, retry _does_ provide the normal err and result. so it has to be adapted to not send the second parameter.